projects/jdwp/codegen/dataclass_generator.py

```diff
@@ -44,6 +49,10 @@ def __get_python_type_for(self, struct: Struct, field: Field) -> str:
             case _:
                 return python_type_for(type)
 
+    def __get_element_struct_name(self, field: Field) -> str:
+        element_type = field.type.element_type
+        return self.__struct_to_name[element_type]
+
```

Comment:
Does this type check ? I think we either should change the type of field to `Field[Array]` or assert something about the type of `field.type`